### PR TITLE
Add Addin API doc

### PR DESCRIPTION
**Pull request by RongqiZ**
## Category

- [ ] Content fix
- [x] New article

## Related issues

None

## What's in this Pull Request?

Added one doc about Add-in APIs. They could help to have a clear view and better management of the Add-ins in tenant level, including these:
- Get available Add-ins in sites
- Get Add-in permissions in site collections
- Get tenant ACS service principals
- Get Add-in principals in site collections
- Uninstall Add-ins
- Get uninstall Add-in job status

### CODE REVIEW

File: docs/apis/addin-management-apis.md [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+description: We provided some Add-in related APIs to get Add-in's installation info and uninstall the Add-in. 
+audience: admin 
+ms.date: 03/25/2023 
+ms.localizationpriority: medium
+---
+
+# SharePoint Add-in Management APIs
+This documentation will introduce some APIs which related to SharePoint Add-ins. They could help to have a clear view
+and better management of the Add-ins in tenant level, including these:
+- Get available Add-ins in sites
```
Comment (andrewconnell) [inline docs/apis/addin-management-apis.md:12]: Please add line breaks around bullet lists.

File: docs/apis/addin-management-apis.md [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+| allowAppOnly                    | bool                                   | This identifies if the Add-in allows app only mode. For more information, see [Add-in authorization policy types in SharePoint](../sp-add-ins/add-in-authorization-policy-types-in-sharepoint.md). |
+
+#### SPTenantScopedPermissionInfo
+
+| Name    | Type   | Description                                                                                                                                                                                         |
+|---------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| feature | string | The feature name of the permissions, it should be one of these:<br/> <li> Taxonomy <br/> <li> Social <br/> <li> ProjectServer <br/> <li> Search <br/> <li> BcsConnection <br/> <li> Content         |
```
Comment (andrewconnell) [inline docs/apis/addin-management-apis.md:137]: This isn't a valid bullet list. Remove the `<br />` tags and surround all `<li>` with the correct `<ul>` element.

**Review by andrewconnell — CHANGES_REQUESTED** [review]

File: docs/apis/addin-management-apis.md (hunk 2 above)
Comment (RongqiZ) [reply docs/apis/addin-management-apis.md]: Updated, thanks!

File: docs/apis/addin-management-apis.md [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+- Get Add-in permissions in site collections
+- Get tenant ACS service principals
+- Get Add-in principals in site collections
+- Uninstall Add-ins
+- Get uninstall Add-in job status
+
+For more information about SharePoint Add-in, see [SharePoint Add-ins](../sp-add-ins/sharepoint-add-ins.md).
+
+## Prerequisites
+- App-only mode token. [Get token example](https://github.com/pnp/pnpcore/blob/dev/docs/polyglot/Getting%20started%20-%20application%20permissions.ipynb)
```
[the remark this answers is not in the document]
Comment (RongqiZ) [reply docs/apis/addin-management-apis.md:25]: Updated, thanks!

File: docs/apis/addin-management-apis.md (hunk 1 above)
Comment (RongqiZ) [reply docs/apis/addin-management-apis.md:14]: Updated, thanks!

**Review by andrewconnell — APPROVED** [review]

**Comment by VesaJuvonen** [comment]
thx @RongqiZ for this - will get published today as part of other docs. Great work on this! 👏👏👏

**Pull request merged**